ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -1998,6 +1998,43 @@ def test_paddle_rsqrt(
     )
 
 
+# scale
```

Comment:
There are some linting issues in this file. Please resolve them. 😄 You should use pre-commit!

Reply:
I used pre commit and in local space I resolved those lint errors as well. I dont how in the CI its showing the lint errors.